app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -235,6 +236,41 @@ public void onFailure(@NonNull Exception e) {
                 Toast.makeText(ActivityDescriptionActivity.this, e.getMessage(), Toast.LENGTH_SHORT).show();
                 Log.d(TAG, "An error occurred! Participant may be already registered in Firebase Firestore!");
             }
+        } else {
+            Toast.makeText(ActivityDescriptionActivity.this, "Already registered!", Toast.LENGTH_SHORT).show();
+        }
+    }
+
+    /**
+     * Registering user from the activity document on Firebase Firestore (field array "participantId")
+     */
+    public void unregisterFromActivity(View v) {
```

Comment:
Good ! So this function will be useful for delete an account

Reply:
Well not really. This method can be used by the participant to unregister from the activity (in case he's already registered and he's NOT the organiser of the activity). The"delete account feature" is implemented in the `EditProfileActivity`.